lib/msal-browser/src/app/ClientApplication.ts

```diff
@@ -49,6 +49,9 @@ export abstract class ClientApplication {
     // Flag to indicate if in browser environment
     protected isBrowserEnvironment: boolean;
 
+    // Sets the account to use if no account info is given
+    private activeAccountId: string | null;
```

Comment:
I think we shuold consider making this name reflect which id is used:

```suggestion
    private activeLocalAccountId: string | null;
```

Reply:
done.